src/orchestrationapi/orchestration.go

```diff
@@ -115,6 +122,9 @@ type OrchestrationBuilder struct {
 
 	isSetClient bool
 	clientAPI   client.Clienter
+
+	isSetStorageDriver bool
+	storageDriverIns   storagedriver.StorageDriver
```

Comment:
The orchestrationapi is not the storagemgr, we should implement storagemgr in src/controller/storagemgr/.
Please refer to the [discoverymgr](https://github.com/lf-edge/edge-home-orchestration-go/blob/master/src/controller/discoverymgr/discovery.go) when you implement the storagemgr.